.ibm/pipelines/kubernetes-tests.sh

```diff
@@ -16,6 +16,7 @@ export SKIP_USER_LOGIN_TESTS=true
     echo Using Devfile proxy: ${DEVFILE_PROXY}
     make install
     make test-integration-cluster
+    make test-doc-automation
```

Comment:
@feloy why do we need to run `make test-doc-automation` on both file?

Reply:
No, I'm checking on which one it passes. I'll remove one of them when the other one passes

Reply:
Understood, Thanks Philippe